### ISSUE TITLE
Supabase GetMany only runs once even with multiple inputs

### ISSUE THREAD

**Issue by apertureless** [n8n-io/n8n#11108]
### Bug Description

I have two supabase nodes.

1. Node: Get Many Rows with some filters. Output are for example 3 entries (users)
2. Node: Again a Get Many Rows. Using {{user.id}} expression to get the corresponding profiles with EQUALS filter.

There is a tooltip claiming "This node runs multiple times, once for each input item. Use ‘Execute Once’ in the node settings if you want to run it only once."

However the output is always 1 item. Also further nodes always get only 1 item.

### To Reproduce

1. Create Supabase node -> Row -> Get Many -> Make a query that returns multiple items
2. Create Supabase node -> Row -> Get Many -> Make query with input value from previous node

### Expected behavior

Second node should run multiple times to process all inputs.

### Operating System

hosted

### n8n Version

hosted

### Node.js Version

hosted

### Database

PostgreSQL

### Execution mode

main (default)

![2024-10-04_15-44](https://github.com/user-attachments/assets/cdceb737-f068-4d70-9a1a-2065cfdfe676)

**Comment by Joffcom** [comment]
Hey @apertureless,

We have created an internal ticket to look into this which we will be tracking as "GHC-284"

**Comment by Joffcom** [comment]
We’re taking some time to review and tidy up our issue tracker. It looks like this issue hasn’t had any recent activity for the past 6 months. To keep our backlog focused on current priorities, we’re closing this issue for now.

That said, this doesn’t mean the issue is forgotten! If this is still relevant or you have more information to add, feel free to reopen it or create a new issue and we’d be happy to revisit it.

Thank you for understanding and for contributing to the project!
